src/pages/ContactGroup.tsx

```diff
@@ -0,0 +1,143 @@
+          sx={{
+            fontSize: ['2.2rem', '2.5rem'],
+            textAlign: 'center',
+          }}
+        >
+          Jeu De Boules Met Vrienden
+        </Typography>
+      </Box>
+
+      <Box sx={{ marginInline: [3, 15] }}>
+        <Typography
+          variant="h6"
+          component="p"
+          sx={{
+            fontSize: '1rem',
+            lineHeight: 1.6,
+          }}
+        >
+          Wil je kennis komen maken met het spel Jeu de Boules? Dit kan met
+          vrienden, kennissen, collega’s of wie dan ook. Ervaren spelers zullen
+          het spel uitleggen en jullie begeleiden. Onze vrijwilligers kunnen
+          jullie voorzien van koffie, een drankje en als jullie willen een
+          hapje.
+        </Typography>
+      </Box>
+
+      <Box className="flex-center" sx={{ flexDirection: 'column', margin: 2 }}>
+        <LineDivider />
+
+        <Box>
+          <img src={groupImg} alt="groep foto" className="img" />
+        </Box>
+
+        <LineDivider />
+      </Box>
+
+      <Typography
+        variant="h4"
+        component="h2"
+        margin={3}
+        sx={{
+          fontSize: ['1.4rem', '1.3rem'],
+          wordWrap: 'break-word',
+          whiteSpace: 'normal',
+          textAlign: 'center',
+        }}
+      >
+        Voor meer informatie kun je geheel vrijblijvend contact opnemen met:
+      </Typography>
+
+      <Box
+        className="contact-info"
+        sx={{
+          display: 'flex',
+          flexDirection: ['column', 'row'],
+          gap: 3,
+          justifyContent: 'center',
+          alignItems: 'center',
+        }}
+      >
+        <Typography
+          variant="h5"
+          component="p"
+          sx={{
+            fontSize: ['1rem', '1.3rem'],
+            borderRight: ['none', '3px solid #ff6700'],
+            paddingRight: [0, 5],
+          }}
+        >
+          Desire{' '}
+          <PhoneIcon
+            className="icon"
+            sx={{ width: ['25px', '40px'], height: ['25px', '40px'] }}
+          />{' '}
+          0653802691
+        </Typography>
+
+        <Typography
+          variant="h5"
+          component="p"
+          sx={{
+            fontSize: ['1rem', '1.3rem'],
+          }}
+        >
+          Chris{' '}
+          <PhoneIcon
+            className="icon"
+            sx={{ width: ['25px', '40px'], height: ['25px', '40px'] }}
+          />{' '}
+          0623430003
+        </Typography>
+      </Box>
+
+      <Typography
+        variant="h6"
+        component="p"
+        margin={4}
+        sx={{
+          fontSize: ['1rem', '1.2rem'],
+          wordWrap: 'break-word',
+          whiteSpace: 'normal',
+          textAlign: 'center',
+        }}
+      >
+        of stuur een mail naar:
+        <Box>
+          <MailOutlineIcon
+            className="icon"
+            sx={{ width: ['25px', '40px'], height: ['25px', '40px'] }}
+          />
+          <a href="#">promotievdsport@jbvdewalnoot.nl</a>
```

Comment:
You should be avaible when clicked on this a tag, it should take you to a mail 
```suggestion
          <a href="mailto:promotievdsport@jbvdewalnoot.nl">promotievdsport@jbvdewalnoot.nl</a>
```